lib/car/car.worker.ts

```
@@ -0,0 +1,322 @@
+  getIntersection,
+  Intersection,
+  lerp,
+} from "@/utils/math";
+import { Edge } from "@/lib/primitives/edge";
+import { Node } from "@/lib/primitives/node";
+import { Polygon } from "@/lib/primitives/polygon";
+import { EdgeJson, PolygonJson } from "@/types/save";
+
+let carState: WorkerCarState;
+
+onmessage = (event: MessageEvent<CarWorkerInboundMessage>) => {
```

Biome (2.3.14)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)